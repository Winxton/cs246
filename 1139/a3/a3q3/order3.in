! a
+ a c
+ a b
+ c d
+ c e
+ c f
+ b g
+ e i
+ e h
p
? i
