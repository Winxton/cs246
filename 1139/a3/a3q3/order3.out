a is parent of c,b
c is parent of d,e,f
d has no children
e is parent of i,h
i has no children
h has no children
f has no children
b is parent of g
g has no children
a,c,e,i
