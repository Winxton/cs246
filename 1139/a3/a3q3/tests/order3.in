! a
+ a c
+ a b
+ c d
+ c e
+ c f
+ b g
+ e i
+ e h
p
? i
* d
+ c d
p
* c
? d
? e
? i
? h
+ i p
+ b i
! h
p
