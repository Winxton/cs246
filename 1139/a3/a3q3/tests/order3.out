a is parent of c,b
c is parent of d,e,f
d has no children
e is parent of i,h
i has no children
h has no children
f has no children
b is parent of g
g has no children
a,c,e,i
a is parent of c,b
c is parent of e,f,d
e is parent of i,h
i has no children
h has no children
f has no children
d has no children
b is parent of g
g has no children
Not found
Not found
Not found
Not found
Failed
h is parent of a
a is parent of b
b is parent of g,i
g has no children
i has no children
